mkosi/manifest.py

```diff
@@ -231,13 +243,22 @@ def as_dict(self) -> dict[str, Any]:
         if self.context.config.release is not None:
             config["release"] = self.context.config.release
 
+        if self.context.config.output_format == OutputFormat.sysext:
+            config["output_format"] = "sysext"
+        elif self.context.config.output_format == OutputFormat.confext:
+            config["output_format"] = "confext"
+        else:
+            config["output_format"] = "image"
```

Comment:
Please simplify this to

```
config["output_format"] = self.context.config.output_format.value
```

Reply:
I moved it into the config dict definition itself.

Done